go-controller/vendor/github.com/ovn-org/libovsdb/client/client.go

```diff
@@ -778,7 +778,10 @@ func (o *ovsdbClient) transact(ctx context.Context, dbName string, operation ...
 	if o.rpcClient == nil {
 		return nil, ErrNotConnected
 	}
-	o.logger.V(4).Info("transacting operations", "database", dbName, "operations", fmt.Sprintf("%+v", operation))
+	dbgLogger := o.logger.WithValues("database", dbName).V(4)
+	if dbgLogger.Enabled() {
+		dbgLogger.Info("transacting operations", "operations", fmt.Sprintf("%+v", operation))
+	}
```

Comment:
Not relevant to this PR but... gonna ask anyway

the logger already checks internally if it is enabled or not, it is a pity that have to do it ourselves to avoid the `Sprintf` which is a discourage use the the logging framework anyway. What would we lose if we just did:
```
dbgLogger.Info("transacting operations", "operations", operation))
``` 

AFAIK the logger should already be doing a decent job printing stuff, including fields names of structs, etc.

Reply:
@jcaamano I didn't want to change log format from what we were already printing; I did try to see if the logger was just doing "%+v" anyway but the internals are pretty obscure.

I think we should do this upstream in libovsdb though and let it filter back into 4.13+?

Reply:
Yeah, already doing it in on one of my PRs.

It should be happening around here https://github.com/go-logr/logr/blob/a49c9717d660988d99b6f931e67c49745caacbcd/funcr/funcr.go#L493